buildsystem/modules/FindPython.cmake

```diff
@@ -169,6 +170,19 @@ foreach(PYTHON ${PYTHON_INTERPRETERS})
 		find_library(PYTHON_LIBRARIES ${PYTHON_LIBRARY_NAME}
 			PATHS "${PYTHON_LIBRARY_DIR}" "${PYTHON_LIBPL}"
 		)
+
+        # On macOS, the above was not working for homebrew installations.
+        # Insetad we inqurie `python3-config --prefix` for the installation,
+        # and find the location of the libraries.
+        if (PYTHON_LIBRARIES STREQUAL "PYTHON_LIBRARIES-NOTFOUND")
+            py_exec(
+                "from distutils.sysconfig import PREFIX; print(PREFIX)"
+                PYTHON_CONFIG_DIR
+            )
+            # TODO: Don't hardcode version numbers
+            set(PYTHON_LIBRARIES "${PYTHON_CONFIG_DIR}/lib/libpython3.6.dylib")
```

Comment:
Please fix the indentation of this block (with tabs).
Why is the hardcoding neccessary?
I would suggest to investigate why `${PYTHON_LIBRARY_NAME}` is not found in `PYTHON_LIBRARY_DIR` where python distutils sysconfig tells us about.

 (also, typos in `insetad`, `inqurie`).